docs/sql_reference/src/asciidoc/_chapters/sql_language_elements.adoc

```diff
@@ -3352,6 +3354,60 @@ The escape character indicates that the underscore in 'FLOPPY_DISK' is
 part of the string to search for, not a wild-card character.
 
 <<<
+[[regexp_predicate]]
+=== REGEXP Predicate
+
+Performs a pattern match of a string expression against a pattern . 
+The pattern can be an extended regular expression. 
+Returns 1 if expression matches pattern; otherwise it returns 0. 
+If either expression or pattern is NULL, the result is NULL.
+
+[[regexp_syntax]]
+==== Syntax
+```
+match-value [NOT] REGEXP pattern
+```
+
+* `_match-value_`
++
+is a character value expression that specifies a set of strings to
+search for that match the _pattern_.
+
+* `_pattern_`
++
+is a character value expression that specifies the pattern string for
+the search in form of Regular Expression.
+The Trafodion Regular Expression is following the POSIX regular expression rules.

```

Comment:
Grammar nits. Possible wordsmith: "is a character value expression that specifies a regular expression. Trafodion regular expressions follow POSIX regular expression rules."